src/main/java/de/tum/cit/aet/artemis/tutorialgroup/domain/TutorialGroup.java

```diff
@@ -104,6 +104,18 @@ public class TutorialGroup extends DomainObject {
     @Transient
     private String teachingAssistantName;
 
+    /**
+     * This transient fields is set to the name of the teaching assistant of this tutorial group
+     */
+    @Transient
+    private Long teachingAssistantId;
+
+    /**
+     * This transient fields is set to the name of the teaching assistant of this tutorial group
+     */
+    @Transient
+    private String teachingAssistantImageUrl;
```

Comment:
We want to use DTOs instead of the real data objects in the endpoints. Would it be a big effort to use a DTO here? By adding more transient attributes you only increase the inevitable work to switch to a DTO some day...

Reply:
Hey! Yeah you are right, a refactor to a DTO would be good here. I think it might be a bit of effort, but not too much. For better seperation I'll create a seperate issue and work on that asap 👍